src/pretix/base/metrics.py

```diff
@@ -0,0 +1,150 @@
+import redis
+
+ns_prefix = "pretix_metrics_"
+
+r = redis.Redis(host=redishost, port=redisport)
+
+
+class MetricsError(Exception):
+    def __init__(self, msg):
+        self.msg = msg
+
+
+class Metric(object):
+    """
+    Base Metrics Object
+    """
+
+    def __init__(self, name, helpstring, labelnames=[]):
+        self.name = name
+        self.help = helpstring
```

Comment:
No, the helpstring is not used (yet), but it is good practice to require one.

renamed.